exporter/containerimage/exptypes/types.go

```diff
@@ -23,3 +40,64 @@ type Platform struct {
 	ID       string
 	Platform ocispecs.Platform
 }
+
+func ExporterAnnotationIndexKey(key string) string {
```

Comment:
Drop the `Exporter` prefix for all these. Package already says exporter. We can change the `const` this pkg defines later as well. Afaics they should be `const exptypes.KeyConfigDigest` etc.

You could also move all the new annotation stuff to `annotations.go` in this file instead of `types.go`.